src/beignet/_bisect.py

```diff
@@ -0,0 +1,59 @@
+from dataclasses import dataclass
```

Comment:
I know `dataclass` is a class but I prefer `import dataclasses` and `dataclasses.dataclass`.

Reply:
Unused?